.azure-pipelines/client.yml

```diff
@@ -58,6 +60,7 @@ jobs:
   - job: 'Test'
 
     strategy:
+      maxParallel: $[ variables['MaxParallelTestJobs'] ]
```

Comment:
I assume there is a reason you need to use the special syntax for accessing the variable?

Reply:
The normal syntax `'$(MaxParallelTestJobs)'` doesn't work for this variable, but the alternate syntax does.  I suspect this is because this variable must be expanded very early in the pipeline processing.

Reply:
I could add a comment explaining this if you want, but I think it's obvious we only use this syntax when required.